<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>584</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>794</width>
    <height>584</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1461</width>
    <height>703</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>666</width>
     <height>428</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1461</width>
     <height>643</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCustomPlot" name="serialPlot" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>651</width>
        <height>331</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1421</width>
        <height>421</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="SerialConnectButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SerialRunButton">
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="QuitButton">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="ADC_CH_groupBox">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>111</height>
           </size>
          </property>
          <property name="title">
           <string>ADC Channels</string>
          </property>
          <widget class="QCheckBox" name="checkBox_CH0">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>97</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>CH0</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox_CH1">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>50</y>
             <width>97</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>CH1</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox_CH2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>80</y>
             <width>97</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>CH2</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>291</width>
            <height>141</height>
           </size>
          </property>
          <property name="title">
           <string>Dsiplay Settings</string>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>85</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Max Voltage</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="MaxVoltEdit">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>30</y>
             <width>174</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>174</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>174</width>
             <height>27</height>
            </size>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>V</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>8</x>
             <y>60</y>
             <width>82</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>82</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Min Voltage</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="MinVoltEdit">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>60</y>
             <width>174</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>174</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>174</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>V</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>86</width>
             <height>29</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>86</width>
             <height>29</height>
            </size>
           </property>
           <property name="text">
            <string>Precision:</string>
           </property>
          </widget>
          <widget class="QSlider" name="TimePeriodSlider">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>90</y>
             <width>169</width>
             <height>29</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>169</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>169</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximum">
            <number>6</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>271</width>
            <height>121</height>
           </size>
          </property>
          <property name="title">
           <string>Sampling Settings</string>
          </property>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>84</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Time range: </string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>98</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>98</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Sample speed</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="TimeRangespinBox">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>30</y>
             <width>169</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>169</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>169</width>
             <height>27</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
          <widget class="QLineEdit" name="SamplingTimeEdit">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>60</y>
             <width>169</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>169</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>169</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>sampling in ms</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
